plugins/scanners/scancode/src/main/kotlin/ScanCodeResultModel.kt

```diff
@@ -61,6 +61,7 @@ sealed interface FileEntry {
     val licenses: List<LicenseEntry>
     val copyrights: List<CopyrightEntry>
     val scanErrors: List<String>
+    val matchedText: String
```

Comment:
commit-msg: Please explain the reason for adding this property.

Reply:
I can't recall at the moment; IDE might complain it was missing. But turns out it isn't needed.